tools/salmon-index-tool.cwl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env cwl-runner
+class: CommandLineTool
+id: salmon-index-tool
+label: salmon-index-tool
+cwlVersion: v1.0
+
+requirements:
+  - class: DockerRequirement
```

Comment:
Since you use the new format above.  Maybe change to

```
  DockerRequirement:
    dockerPull: combinelab/salmon
```
Semantics...

Reply:
I think i did.